presto-main/src/test/java/com/facebook/presto/operator/scalar/BenchmarkMapConcat.java

```diff
@@ -92,7 +92,7 @@
         private String name = "map_concat";
 
         @Param({"left_empty", "right_empty", "both_empty", "non_empty"})
-        private String mapConfig = "left_empty";
+        private String mapConfig = "non_empty";
```

Comment:
Is this change intended?

Reply:
Yes it is inteded. left_empty means the left input array is empty and is not the case we cared most.